components/Banner/Banner.tsx

```diff
@@ -13,8 +13,11 @@ export const Banner = ({ src, title }: Props) => {
         alt={`${title} - banner`}
         priority
         placeholder="blur"
-        objectFit="cover"
-        layout="fill"
+        fill
+        sizes="100vw"
```

Comment:
so I guess `styles.banner` don't need `100vw` style

Reply:
I left `100vw` in the `styles.banner` because without it doesn't work but I removed `sizes` from the `Image` component to don't duplicate styles. If you have any idea how it should look please let me know